models/classes/user/events/LtiUserCreatedEvent.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019  (original work) Open Assessment Technologies SA;
+ *
+ * @author Oleksandr Zagovorychev <zagovorichev@1pt.com>
+ */
+
+namespace oat\taoLti\models\classes\user\events;
+
+
+class LtiUserCreatedEvent extends LtiUserEvent
+{
+    const EVENT_NAME = __CLASS__;
+
+    public function getName()
+    {
+        return self::EVENT_NAME;
```

Comment:
Why can't we return `__CLASS__` here directly, without class constant?

Reply:
because `getName` uses from the object and const we use when we attach triggers, you could check it on your own, couldn't you =) ?